triage/issue-tracker.rst

```diff
@@ -84,16 +84,43 @@ The submission form has only two fields that you need to fill:
   information as appropriate). In particular, *what version of Python* you
   were using.
 
-Understanding the issue's progress and status
----------------------------------------------
+It is also acceptable to tag someone to if you think the issue
+should be brought to their attention.  Use the :ref:`experts` to know
+who wants to be added to the nosy list for issues targeting specific areas.
 
 There is a number of additional fields like **Assignees**, **Labels**,
 **Projects**, and **Milestone**. Those are filled by triagers and core
-developers, this is covered in the :ref:`triaging` page. You don't need
+developers and are covered in the :ref:`triaging` page. You don't need
```

Comment:
Is it, though? Right now, the only mention I see of how to use these fields on that page is similarly brief:

> Where appropriate, set the `Assignees`, `Reviewers`, `Project` fields, and possibly `@mention` relevant people.

To note, some of them (particularly `Assignees`, which seems to be the one with the most confusion and least clarity or constancy about how to use it among many triagers and even core developers) are already described on the previous `labels` page, which could potentially be moved over to the `triaging` page (but are currently eliminated entirely without apparent replacement in this PR).

Reply:
The `experts` page (linked in the previous paragraph) covers who can be assigned too, so maybe we could tweak that sentence to say "...to be tagged *or assigned*"?
We could probably drop Milestones from the list since we don't use them.

Reply:
Yeah, makes sense to me I guess :+1: